docs/source/actor-rhel7-o-rhel8.md

```diff
@@ -0,0 +1,315 @@
+# How to create a Leapp actor for RHEL 7 to 8 upgrade
+
+## Introduction
+
+This document is intended for all people who want to contribute to RHEL upgrade process using Leapp tool. After reading through this document, you will be able to transform your expertise in certain parts of RHEL into improvements of the RHEL 7 to 8 upgrade tooling. You are expected to be familiar with the concept of in-place upgrades and ideally to have at least high-level knowledge about the Leapp tool.
+
+## Setting up the development environment
+
+Leapp actors are written in Python, so your usual Python development setup can be used during the process of creating a new actor.
+
+## Tools
+
+The main tools you will use for the actor development are listed below.
+
+### leapp
+
+The leapp framework uses provides the libraries required to be imported by any actor and also a binary tool used to control the execution of actors within a workflow.  
+
+### snactor
```

Comment:
As a newcomer I might like to see the source code for the tool, can you add a link?